.eslintrc.js

```diff
@@ -11,7 +11,13 @@ module.exports = {
     es6: true,
     jest: true,
   },
-  extends: ['airbnb', 'prettier', 'prettier/react'],
+  extends: [
+    'plugin:react/recommended',
```

Comment:
Could you send me a link with more information about that plugin:react/recommended "code style"? And I am curious why you decided to change it?

Reply:
Have a look:
https://www.npmjs.com/package/eslint-plugin-react

I had some issues with previous configuration and realized that the `airbnb` config doesn't really do much for us (and it seems it is less popular recently). Then I decided to rebuild the configuration pretty heavily :smile: For example we didn't have any good TypeScript configuration before.